docs/content/how-to/system-tables.md

```diff
@@ -198,12 +198,12 @@ and some historical information of the snapshots. You can also get all tag names
 SELECT * FROM MyTable$tags;
 
 /*
-+----------+-------------+-----------+-------------------------+--------------+
-| tag_name | snapshot_id | schema_id |             commit_time | record_count |
-+----------+-------------+-----------+-------------------------+--------------+
-|     tag1 |           1 |         0 | 2023-06-28 14:55:29.344 |            3 |
-|     tag3 |           3 |         0 | 2023-06-28 14:58:24.691 |            7 |
-+----------+-------------+-----------+-------------------------+--------------+
++----------+-------------+-----------+-------------------------+--------------+--------------+
+| tag_name | snapshot_id | schema_id |             commit_time | record_count |   branches   |
++----------+-------------+-----------+-------------------------+--------------+--------------+
+|     tag1 |           1 |         0 | 2023-06-28 14:55:29.344 |            3 |      []      |
```

Comment:
```suggestion
|     tag1 |           1 |         0 | 2023-06-28 14:55:29.344 |            3 |      [main]      |
```
How about use `main` ?

Reply:
I think it's better to given empty branches instead of `main` if there is no branch for the tag